py/tests/unit/with_runtime_sparkling/test_Spark_to_H2O_conversions.py

```diff
@@ -277,7 +277,7 @@ def testConvertTimeValueFromSparkToH2OAndBack(spark, hc, timeZone, sparkType):
 
     hf = hc.asH2OFrame(df)
     hfResultString = hf.__unicode__()
```

Comment:
why do you use `__unicode__()` here? This is a remnant from Py2.7 and is obsolete. Is SW also supporting Py2.7?
FYI, `hf.to_str()` should now provide a correct unicode representation in all versions of Py

Reply:
> why do you use __unicode__() here?

I'm not the author of this test

> Is SW also supporting Py2.7?

Yes,  for Spark 2.2 - 3.0

Reply:
I've replaced `hf.__unicode__()` with `hf.to_str()`